util/src/main/java/care/smith/fts/util/auth/HttpClientOAuth2Auth.java

```diff
@@ -0,0 +1,35 @@
+package care.smith.fts.util.auth;
+
+import care.smith.fts.util.auth.HttpClientOAuth2Auth.Config;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.context.annotation.Conditional;
+import org.springframework.security.oauth2.client.ReactiveOAuth2AuthorizedClientManager;
+import org.springframework.security.oauth2.client.web.reactive.function.client.ServerOAuth2AuthorizedClientExchangeFilterFunction;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.client.WebClient.Builder;
+
+/** OAuth2 Authentication using oauth2 client credentials flow. */
```

Comment:
Does this implementation really care, that we use client credentials flow?

Reply:
Absolutely not, who wrote that?^^